runtime/default.policy

```diff
@@ -464,6 +464,9 @@ rules:
       && open.flags & OPEN_WRITE_FLAGS > 0
       && process.file.path not in SYSTEM_PACKAGE_MANAGERS
       && process.ancestors.file.path not in SYSTEM_PACKAGE_MANAGERS
+  - id: selinux_disable_enforcement
```

Comment:
Are we sure we want this rule ? I see that this rule was added/removed multiple times in the past weeks

Reply:
I'm not sure tbh, it's what the generation script created. Not sure who the owner of this is but if this is not needed, then it should be removed from internal repo too.

Reply:
I think part of the problem is that this SElinux rule was only present in security-agent-policies and not in security-monitoring. So when we did a sync, the one in security-agent-policies was removed.
So I think it should be in security-agent-policies. If it causes any issue, we can remove in security-monitoring and sync again

Reply:
Ok let's merge then